docs/samples/Microsoft.ML.Samples/Dynamic/LightGBM/LightGBMBinaryClassification.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using Microsoft.ML.Data;
```

Comment:
>Microsoft [](start = 6, length = 9)

We already have Transforms folder, I'm creating Trainers folder with Recommendation subfolder in [https://github.com/dotnet/machinelearning/pull/2451](https://github.com/dotnet/machinelearning/pull/2451)
maybe it's worth to put them in Trainers/BinaryClassification, Trainers/MultiClassification, Trainers/Regression?

Overwise it start look like trash pile :)

Reply:
+1 for the Trainers folder. 

---
In reply to: [255223584](https://github.com/dotnet/machinelearning/pull/2476#discussion_r255223584) [](ancestors = 255223584)